cmake/common.cmake

```diff
@@ -524,6 +524,7 @@ if(NOT ${TARGET} STREQUAL "virtual")
     ${CMAKE_THREAD_LIBS_INIT}
     ccfcrypto.host
     evercrypt.host
+    curl
```

Comment:
Do we need to update https://github.com/microsoft/CCF/blob/master/getting_started/setup_vm/roles/ccf_dependencies/vars/common.yml to install curl as a dependency? Also perhaps add it here: https://github.com/microsoft/CCF/blob/master/sphinx/source/getting_started.rst#details?

Reply:
Good catches. It seems its installed already in our image, but I've added it for safety.

Reply:
Might be a good opportunity to start using find_package()?